ipi/engine/forcefields.py

```diff
@@ -174,12 +168,16 @@ def queue(self, atoms, cell, reqid=-1):
     def poll(self):
         """Polls the forcefield object to check if it has finished."""
 
-        for r in self.requests:
-            if r["status"] == "Queued":
-                r["t_dispatched"] = time.time()
-                r["result"] = [0.0, np.zeros(len(r["pos"]), float), np.zeros((3, 3), float), ""]
-                r["status"] = "Done"
-                r["t_finished"] = time.time()
+        self._threadlock.acquire()
```

Comment:
ok, this is also in the other pull request.

Reply:
yep now it's merged